projects/igniteui-angular/src/lib/grids/grid/grid.search.spec.ts

```diff
@@ -333,7 +265,6 @@ describe('IgxGrid - search API #grid', () => {
 
             grid.sort({ fieldName: 'JobTitle', dir: SortingDirection.Asc, ignoreCase: true });
             fix.detectChanges();
-
             activeHighlight = rv.querySelector('.' + component.activeClass);
             let highlights = rv.querySelectorAll('.' + component.highlightClass);
```

Comment:
Can you create 2 constants for active and  highlight classes and do not get them from the component.